docs/docs/1.2.3/policies/mutual-tls.md

```diff
@@ -284,6 +283,10 @@ mtls:
 :::
 ::::
 
+### Intermediate CA
+
+It is possible to use Intermediate CA with Provided backend. In `cert` section, put Intermediate CA first and then Root CA. Separate those certificates by an empty line. 
+
```

Comment:
I'd like to copyedit this too, but a more pressing question: would this section be more useful not as a subsection but as part of the intro to the Provided CA section? (If yes, I'm happy to push changes unless you'd rather have suggestions.)